docs/config.md

```diff
@@ -367,14 +367,17 @@ Before proceeding, it will be helpful to see the documentation on [Model Analyze
 
 ### `<constraint>`
 
-A constraint, specifies the bounds that determine a successful run. There are
-three constraints allowed:
-
-| Option Name        |   Units   | Constraint | Description                                          |
-| :----------------- | :-------: | :--------: | :--------------------------------------------------- |
-| `perf_throughput`  | inf / sec |    min     | Specify minimum desired throughput.                  |
-| `perf_latency_p99` |    ms     |    max     | Specify maximum tolerable latency or latency budget. |
-| `gpu_used_memory`  |    MB     |    max     | Specify maximum GPU memory used by model.            |
+A constraint, specifies the bounds that determine a successful run. The table below shows examples
+of the types of constraints allowed:
+
+| Option Name               |   Units   | Constraint | Description                                            |
+| :------------------------ | :-------: | :--------: | :----------------------------------------------------- |
+| `perf_throughput`         | inf / sec |    min     | Specify minimum desired throughput.                    |
+| `perf_latency_p99`        |    ms     |    max     | Specify maximum tolerable latency or latency budget.   |
+| `output_token_throughput` | inf / sec |    min     | Specify minimum desired output token throughput.       |
+| `inter_token_latency_p99` |    ms     |    max     | Specify maximum tolerable input token latency.         |
+| `time_to_first_token_p99` |    ms     |    max     | Specify maximum tolerable time to first token latency. |
```

Comment:
this lists p99, but technically they could do any of the pXX metrics that MA has, right?

Reply:
That's correct.